frontend/src/scenes/experiments/experimentsLogic.ts

```diff
@@ -34,9 +34,6 @@ export const experimentsLogic = kea<experimentsLogicType>({
             [] as Experiment[],
             {
                 loadExperiments: async () => {
-                    if (!values.hasAvailableFeature(AvailableFeature.EXPERIMENTATION)) {
-                        return []
-                    }
```

Comment:
Was the same problem as with redirecting to the experiments list. Let me know if the experiments endpoint is unavailable in some cases though!